chrysom/client.go

```diff
@@ -354,28 +370,56 @@ func (c *Client) Start(ctx context.Context) error {
 		return ErrUndefinedIntervalTicker
 	}
 
+	c.observer.mu.Lock()
+	defer c.observer.mu.Unlock()
+
+	if c.observer.state == running {
```

Comment:
If you're intending to use this method as an uber/fx Lifecycle hook, it should return an error when `Start` has already been called without previous call to `Stop`.